src/analytics/task/useTaskAnalytics.ts

```diff
@@ -64,6 +64,10 @@ type Action =
       parsleyAvailable: boolean;
       fileName: string;
     }
+  | {
+      name: "Click Task File Parsley Link";
+      fileName: string;
```

Comment:
I suppose it can be gleaned from the URL, but do you think project name would be useful to track here? Perhaps for all of these events.

Reply:
Great idea! Done